packages/angular-material/test/boolean-control.spec.ts

```diff
@@ -22,15 +22,73 @@
   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
   THE SOFTWARE.
 */
-import { async, TestBed } from '@angular/core/testing';
-import { MatCheckboxModule, MatFormFieldModule } from '@angular/material';
 import { NgRedux } from '@angular-redux/store';
 import { MockNgRedux } from '@angular-redux/store/testing';
-import { BooleanControlRenderer } from '../src';
+import { DebugElement } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { MatCheckbox, MatCheckboxModule, MatError, MatFormFieldModule } from '@angular/material';
+import { By } from '@angular/platform-browser';
+import { NOT_APPLICABLE } from '@jsonforms/core';
+import { BooleanControlRenderer, booleanControlTester } from '../src';
 
+const uischema = {
+    type: 'Control',
+    scope: '#/properties/foo'
+};
+
+describe('Material boolean field tester', () => {
+    it('should fail', () => {
```

Comment:
I don't think we need the 'it should fail' spec, as we already test these in `testers.test.ts`

Reply:
it makes sure that the wiring was done correctly, but I'm all for removing those tests, the hard question is what do we need to retest when testing wiring and what not